admin_guide/limit_pod_access_egress.adoc

```diff
@@ -0,0 +1,104 @@
+[[admin-guide-limit-pod-access-egress]]
+= Limit Pod Access with Egress Firewall
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As an {product-title} cluster administrator, you can use egress policy to limit
+the addresses that some or all pods can access from within the cluster, so that:
+
+- A pod can only talk to internal hosts, and cannot initiate connections to the
+public Internet.
++
+Or, 
+- A pod can only talk to the public Internet, and cannot initiate connections to
+internal hosts (outside the cluster).
++
+Or,
+- A pod cannot reach specified internal subnets/hosts that it should have no
+reason to contact.
+
+For example, you can configure projects with different egress policies, allowing
+`<project A>` access to a specified IP range, but denying the same access to
+`<project B>`. 
+
+[CAUTION]
+====
+You must have the 
+xref:../install_config/configuring_sdn.adoc#install-config-configuring-sdn[*ovs-multitenant* plug-in] enabled in order to limit pod access via egress policy.
+====
+
+Project administrators can neither create `*EgressNetworkPolicy*` objects nor
+edit the ones you create in their project. 
+
+The `default` project (and any other global namespace) cannot have egress
+policy. 
+
+[NOTE]
+====
+- If you merge two projects together (via `oadm pod-network join-projects`), 
+then you cannot use egress policy in _any_ of the joined projects.
+
+- If you make a project global (via `oadm pod-network make-projects-global`), 
+then it cannot have an `*EgressNetworkPolicy*`. 
+
+- If an allowed network overlaps with a denied network, then the rules are 
+checked in order, and the first one that matches is enforced.
+====
```

Comment:
I'm not sure if the bullets above are needed. There's no 'introduction' sentence there to link the points. I think it works as three separate sentences, or paras if you want to keep it that way.